### PR TITLE
fix(cloudflare-workers): update wrangler dep

### PR DESCRIPTION
**Pull request by Jolg42**
Because https://github.com/prisma/ecosystem-tests/actions/runs/4934286664/jobs/8819177027#step:8:153
```
 .../node_modules/@cloudflare/wrangler postinstall$ node ./install-wrangler.js
.../node_modules/@cloudflare/wrangler postinstall: Downloading release from https://workers.cloudflare.com/get-npm-wrangler-binary/1.21.0/x86_64-unknown-linux-musl
.../node_modules/@cloudflare/wrangler postinstall:                       ⛔   DEPRECATION   ⛔
.../node_modules/@cloudflare/wrangler postinstall: The version of wrangler you are using is now deprecated.
.../node_modules/@cloudflare/wrangler postinstall: Please update to the latest version of wrangler to prevent critical errors.
.../node_modules/@cloudflare/wrangler postinstall: Run `npm uninstall -g @cloudflare/wrangler && npm install -g wrangler` to update to the latest version
```

### CODE REVIEW

**Comment by Jolg42** [comment]
TODO: currently fails with 
```
✘ [ERROR] Missing entry-point: The entry-point should be specified via the command line (e.g. `wrangler publish path/to/script`) or the `main` config field.
```

**Comment by Jolg42** [comment]
Now fails with, like expected from https://github.com/prisma/prisma/issues/19174 
```
 ✘ [ERROR] Could not resolve "os"

    node_modules/.pnpm/@prisma+client@4.15.0-dev.8_prisma@4.15.0-dev.8/node_modules/@prisma/client/runtime/edge.js:10:3383:
      10 │ ...t";d();f();m();var Rp=require("os"),ia=jo(),Ue=oa(),{env:ge}=w,...
```

File: utils/crypto/envVars.ts [hunk 1]
```diff
@@ -5,9 +5,9 @@ export const envVars = {
     NODEJS_DATA_PROXY_URL: "U2FsdGVkX187qRBSpPYoBu+9kcycYm/7BvLMBftPQbzKid8w0ZXamlDLESDjjlWZZoq40pyYNZxyOa7Y9jXul6U7lf2NPP5EZ41LS1kAW1nUGfSkFf/5Lgek5kbxoaOChuhboHmydnb/Ok5ztaVtLcTzYP4HmSeOAPEwDid9M7sD1CXcsmfA+1bDDpKmggFB",
 
     // dataproxy > cloudflare-workers
-    CF_DATA_PROXY_URL: "U2FsdGVkX187qRBSpPYoBu+9kcycYm/7BvLMBftPQbzKid8w0ZXamlDLESDjjlWZZoq40pyYNZxyOa7Y9jXul6U7lf2NPP5EZ41LS1kAW1nUGfSkFf/5Lgek5kbxoaOChuhboHmydnb/Ok5ztaVtLcTzYP4HmSeOAPEwDid9M7sD1CXcsmfA+1bDDpKmggFB",
-    CF_ACCOUNT_ID: "U2FsdGVkX1/kgKmF6Nc3rIIkben8az6boItdyzPJFj5vEPoxnWtfpyjEO8E6voHLB4Zl/+kddgTA14uboA+40A==",
-    CF_API_TOKEN: "U2FsdGVkX18fx8PpClVvhLV9TuBGm//20tdPUYRowj/hBUw+r7nECbfZ7DBPKgI9lelSXXKOXD4FFNg6FPpURA==",
+    CLOUDFLARE_DATA_PROXY_URL: "U2FsdGVkX187qRBSpPYoBu+9kcycYm/7BvLMBftPQbzKid8w0ZXamlDLESDjjlWZZoq40pyYNZxyOa7Y9jXul6U7lf2NPP5EZ41LS1kAW1nUGfSkFf/5Lgek5kbxoaOChuhboHmydnb/Ok5ztaVtLcTzYP4HmSeOAPEwDid9M7sD1CXcsmfA+1bDDpKmggFB",
+    CLOUDFLARE_ACCOUNT_ID: "U2FsdGVkX1/kgKmF6Nc3rIIkben8az6boItdyzPJFj5vEPoxnWtfpyjEO8E6voHLB4Zl/+kddgTA14uboA+40A==",
+    CLOUDFLARE_API_TOKEN: "U2FsdGVkX18fx8PpClVvhLV9TuBGm//20tdPUYRowj/hBUw+r7nECbfZ7DBPKgI9lelSXXKOXD4FFNg6FPpURA==",
```
Comment (jkomyno) [inline utils/crypto/envVars.ts:10]: Curiosity question: was the env var change necessary just to turn the `CF` string into the more explicit `CLOUDFLARE`?

**Review by jkomyno — APPROVED** [review]

File: utils/crypto/envVars.ts (hunk 1 above)
Comment (Jolg42) [reply utils/crypto/envVars.ts:10]: It triggers a deprecated warning from wrangler CLI

**Pull request merged**